crates/eframe/src/native/wgpu_integration.rs

```diff
@@ -414,7 +421,12 @@ impl<'app> WinitApp for WgpuWinitApp<'app> {
 
     fn suspended(&mut self, _: &ActiveEventLoop) -> crate::Result<EventResult> {
         #[cfg(target_os = "android")]
-        self.drop_window()?;
+        {
+            log::debug!("Android application suspended - asking to save state");
+            self.drop_window()?;
+            Ok(EventResult::Save)
+        }
```

Comment:
I think it makes sense to return `Save` on all platforms here, not just android. That way we should get the same logic on iOS